source/man/doveadm-sync.1.rst.in

```diff
@@ -0,0 +1,357 @@
+   specified log file.
+
+**-s** *previous_state*
+   Use stateful synchronization. If the previous state is unknown, use
+   an empty string. The new state is always printed to standard output.
+
+.. include:: option-u-user.inc
+
+**-x** *mailbox_mask*
+   Exclude the specified mailbox name/mask. The mask may contain "**?**"
+   and "*****" wildcards. The mask can also be a special-use name (e.g.
+   \\Trash). This parameter can be used multiple times.
+
+ARGUMENTS
+=========
+
+*destination*
+   This argument specifies the synchronized destination. It can be one
+   of:
+
+   location
+      Same as *mail_location* setting, e.g. maildir:~/Maildir
+
+   **remote:** *login@host*
+      Uses *dsync_remote_cmd* setting to connect to the remote host
+      (usually via ssh)
+
+   *remoteprefix:login@host*
+      This is the same as remote, except "user@domain\n" is sent before
+      dsync protocol starts. This allows implementing a trusted wrapper
+      script that runs doveadm dsync-server by reading the username from
+      the first line.
+
+   **tcp:** *host[:port]*
+      Connects to remote doveadm server via TCP. The default port is
+      specified by *doveadm_port* setting.
+
+   **tcps:** *host[:port]*
+      This is the same as tcp, but with SSL.
+
+   **command [arg1 [, arg2, ...]]**
+      Runs a local command that connects its standard input & output to
+      a dsync server.
+
+EXIT STATUS
+===========
+
+**dsync** will exit with one of the following values:
+
+**0**
+   Synchronization was done perfectly.
+
+**2**
+   Synchronization was done without errors, but some changes couldn't be
+   done, so the mailboxes aren't perfectly synchronized. Running dsync
+   again usually fixes this. Typically this occurs for message
+   modification sequences with newly created mailboxes. It can also
+   occur if one of the mailboxes change during the syncing.
+
+**1, >2**
+   Synchronization failed.
+
+
+See also :man:`doveadm(1)` for other possible error codes.
+
+EXAMPLE
+=======
+
+SYNCHRONIZATION
+---------------
+
+Synchronize mailboxes with a remote server. Any errors are written to
+stderr.
+
+.. parsed-literal::
+
+      **doveadm sync -u username@example.com remote:server-replica.example.com**
+
+If you need more complex parameters to ssh, you can use e.g.:
+
+.. parsed-literal::
+
+      **doveadm sync -u username@example.com ssh -i id_dsa.dovecot \
+      mailuser@example.com doveadm dsync-server -u username@example.com**
+
+CONVERTING
+----------
+
+Assuming that the *mail_location* setting in
+*@pkgsysconfdir@/conf.d/10-mail.conf* is set to: **mail_location =
+mdbox:~/mdbox**, a logged in system user may convert her/his mails from
+its Maildir in her/his home directory to the mdbox mailbox format. The
+user has to execute the command:
+
+.. parsed-literal::
+
+      **doveadm sync maildir:~/Maildir**
+
+If you want to do this without any downtime, you can do the conversion
+one user at a time. Initially:
```

Comment:
Improperly formatted in manpage after using `.. parsed-literal::` (bold). HTML is fine.